Xamarin.PropertyEditing/OrderedDictionary.cs

```diff
@@ -333,7 +377,11 @@ public bool Remove (TKey key)
 		public void RemoveAt (int index)
 		{
 			TKey key = this.keyOrder[index];
```

Comment:
Should we document the possible out of range exception, like it's done on other methods?